config.json

```diff
@@ -30,6 +30,12 @@
       "values": [],
       "defaultValue": ""
     },
+    "__BlobNamePrefix": {
```

Comment:
I wouldn't create a custom property here for this I would just pass it as a normal msbuild property.